steps/preprocessing.py

```diff
@@ -3,12 +3,20 @@
 
 import numpy as np
 import pandas as pd
+
 from sklearn.externals import joblib
 from sklearn.feature_extraction import text
 import sklearn.preprocessing as sk_prep
-
+from nltk.tokenize import word_tokenize
+from nltk.tokenize import TweetTokenizer
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.corpus import stopwords
 from .base import BaseTransformer
 
+lem = WordNetLemmatizer()
+tokenizer=TweetTokenizer()
+eng_stopwords = set(stopwords.words("english"))
+APPO = {"aren't": 'are not', "can't": 'cannot', "couldn't": 'could not', "didn't": 'did not', "doesn't": 'does not', "don't": 'do not', "hadn't": 'had not', "hasn't": 'has not', "haven't": 'have not', "he'd": 'he would', "he'll": 'he will', "he's": 'he is', "i'd": 'I had', "i'll": 'I will', "i'm": 'I am', "isn't": 'is not', "it's": 'it is', "it'll": 'it will', "i've": 'I have', "let's": 'let us', "mightn't": 'might not', "mustn't": 'must not', "shan't": 'shall not', "she'd": 'she would', "she'll": 'she will', "she's": 'she is', "shouldn't": 'should not', "that's": 'that is', "there's": 'there is', "they'd": 'they would', "they'll": 'they will', "they're": 'they are', "they've": 'they have', "we'd": 'we would', "we're": 'we are', "weren't": 'were not', "we've": 'we have', "what'll": 'what will', "what're": 'what are', "what's": 'what is', "what've": 'what have', "where's": 'where is', "who'd": 'who would', "who'll": 'who will', "who're": 'who are', "who's": 'who is', "who've": 'who have', "won't": 'will not', "wouldn't": 'would not', "you'd": 'you would', "you'll": 'you will', "you're": 'you are', "you've": 'you have', "'re": ' are', "wasn't": 'was not', "we'll": ' will'}
```

Comment:
APPO is quite large dict, thus - for clarity - it should be moved to the utils.py, with more descriptive name.
@shaz13 @jakubczakon what do you guys think?

Reply:
Right now. This is a raw approach. I think this could be handled by a clever regex statement. 
Something that replaces 
``n't --> not,  'd --> would ,  'll --> will , 'm --> am and 's with is`` should cover all words in dict. Let me know what you guys think @kamil-kaczmarek  @jakubczakon

Reply:
@shaz13 I agree, that in principle clever regex would be final goal. However, for the first iteration your dict is good (it is not too large :) ). If you agree, with my idea of small refactor, you can move APPO to `utils.py` in the repo root, (not `steps/utils.py`). If not, let @jakubczakon make final decision on this.